props/configs/src/main/resources/calendar/UTC.calendar

```diff
@@ -1,11 +1,14 @@
 <!--
-  ~ Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+  ~ Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
   -->
 
 <calendar>
     <name>UTC</name>
     <timeZone>UTC</timeZone>
+    <description>UTC</description>
+    <firstValidDate>1500-01-01</firstValidDate>
+    <lastValidDate>2500-12-31</lastValidDate>
```

Comment:
Is this loaded by default? Assuming it will be map w/ ~365,000 elements?

Reply:
It will get loaded if someone uses any calendars.  Good point on the 365k elements.  I have reduced the date range to 73k. 
```
    <firstValidDate>1900-01-01</firstValidDate>
    <lastValidDate>2100-12-31</lastValidDate>
```
There are tradeoffs if we go to a fully lazy load, so let's see if this change is sufficient.